cmd/ndm_daemonset/probe/eventhandler.go

```diff
@@ -47,13 +51,53 @@ func (pe *ProbeEvent) addBlockDeviceEvent(msg controller.EventMessage) {
 	}
 	// iterate through each block device and perform the add/update operation
 	for _, device := range msg.Devices {
-		klog.Infof("Processing details for %s", device.UUID)
+		klog.Infof("Processing details for %s", device.DevPath)
 		pe.Controller.FillBlockDeviceDetails(device)
 		// if ApplyFilter returns true then we process the event further
 		if !pe.Controller.ApplyFilter(device) {
 			continue
 		}
-		klog.Infof("Processed details for %s : %s", device.DevPath, device.UUID)
+		klog.Infof("Processed details for %s", device.DevPath)
+
+		// if GPTBasedUUID need to be used, generate the UUID,
+		// if UUID cannot be generated create a GPT partition
+		if pe.Controller.FeatureGates.IsEnabled(features.GPTBasedUUID) {
+			if len(device.Partitions) > 0 {
+				klog.Info("device has partitions. not creating blockdevice resource")
```

Comment:
Should this not be, not creating GPT labels? I'm skeptical as to why this is logged in the fist place. is klog.debug() not more appropriate?

Reply:
I will update the log level to `klog.V(4).Info`, as klog explicity support a debug level.